### PR TITLE
Activity worker sample

### PR DESCRIPTION
**Pull request by cretz**
## What was changed

* Bootstrap project structure
* Add activity worker sample

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem 'temporalio', path: '../temporal-sdk-ruby'
```
Comment (cretz) [inline Gemfile:5]:
```suggestion
gem 'temporalio', :prerelease => true
```

When SDK released

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
+⚠️ UNDER ACTIVE DEVELOPMENT
+
+The Ruby SDK is under active development and has not released a stable version yet. APIs may change in incompatible ways
+until the SDK is marked stable.
+
+## Usage
+
+Prerequisites:
+
+* Ruby 3.1+
```
Comment (Quinn-With-Two-Ns) [inline README.md:14]: Don't you need Go as well to run the only sample right now?

Comment (cretz) [reply README.md:14]: Ah, yes, I will update the sample's README

**Review by Quinn-With-Two-Ns — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (cretz) [reply README.md:14]: Actually, I saw in the README I have `with [Go](https://go.dev/) installed` which I think is good enough (did similar at https://github.com/temporalio/samples-dotnet/tree/main/src/ActivityWorker)

**Pull request merged**